dev-manual/env/compose.rst

```diff
@@ -97,6 +97,20 @@ It's time to use Docker Compose in order to provision our containers:
    # and the AtoM image has to be built.
    docker-compose up -d
 
+All seven docker containers should now be up and running.
+
+.. IMPORTANT::
+
+   If the atom command below to purge the database fails with the message:
+   ``PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted ...``,
+   you can increase the memory allocated to the Atom container by either adding
```

Comment:
Hi @J4bbi ! Thanks so much for this doc update!

I'd suggest capitalizing the `M` in AtoM, but the rest looks good to go!

Reply:
No, this makes sense! Will approve and merge this change!